homeassistant/components/climate/const.py

```diff
@@ -77,7 +77,7 @@ class HVACMode(StrEnum):
 FAN_MIDDLE = "middle"
 FAN_FOCUS = "focus"
 FAN_DIFFUSE = "diffuse"
-
+FAN_QUIET = "quiet"
```

Comment:
@frenck can you expand a bit on how that would work? Are you talking about having "quiet" in ESPHome but not in HomeAssistant?
It seems that @OttoWinter was already happy with that idea: https://github.com/esphome/esphome/pull/3609

Reply:
Add the constant to the ESPHome integration, and not to the climate platform integration.
(You are adding the constant to `components/climate/const.py`, instead, add it to `components/esphome/climate.py`).

Reply:
I'm actually doing both at the moment. Are you saying that the `esphome` component should know about "quiet" but not `climate`? would that work? It seems that integration [need to use to constant from climate](https://github.com/home-assistant/core/pull/74303/files#diff-d335f2f1f824951421ef024c3be5ef2aee032239bc4bbb723b8a5b54814ae889R113). I suppose I could just write `"quiet"` there instead, but is that what you want? Otherwise I can update the doc as well and make it official.

Reply:
> Otherwise I can update the doc as well and make it official.

An change to the main climate integration requires architectural discussions and approval.

Just move the QUIET constant from the `components/climate/const.py` to `components/esphome/climate.py`, that is all that is needed.

Reply:
It looks like the constant was changed to a bare string instead of relocating it.

Please move the constant per the review comment above.

Reply:
@bdraco hmm, it's not clear to me what place and formatting you expect. Could you be more specific?

Reply:
Keep `FAN_QUIET = "quiet"` but put it in `esphome` instead of `climate` and than use the constant

Reply:
@bdraco hopefully I got it the way you expect this time